sources/packages/backend/apps/api/src/route-controllers/confirmation-of-enrollment/models/confirmation-of-enrollment.dto.ts

```diff
@@ -65,3 +65,22 @@ export class COESummaryAPIOutDTO {
   disbursementScheduleId: number;
   disbursementDate: string;
 }
+
+export interface ConfirmEnrollmentOptions {
```

Comment:
Open for discussion but this one maybe should be moved to a `route-controllers/confirmation-of-enrollment/models/confirmation-of-enrollment.models.ts`